uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Step/Step1104.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+class CRM_HRCore_Upgrader_Step_Step1104 {
+  public function apply() {
+    return true;
```

Comment:
@davialexandre it might be possible to just return `true` from the function that calls this, and avoid requiring a return value here. Maybe I'm missing something, but I don't see the point of it, it's like a very old style of PHP which uses return values to indicate errors instead of throwing exceptions.

Reply:
I'm not sure I understand. Are you saying that we can simply not return anything and it will just work or that we need to change something so that it will fail only if an exception is thrown?

Reply:
> that we need to change something so that it will fail only if an exception is thrown?

This one. I don't know about you, but I've never returned false from an upgrader anyway.